Classes/Configuration/BackendConfigurationManager.php

```diff
@@ -66,13 +66,15 @@ public function getCurrentPageId() {
 	 * @return array
 	 */
 	protected function getPrioritizedPageUids() {
-		return array(
+		$pageUids = array(
 			$this->getPageIdFromGet(),
 			$this->getPageIdFromPost(),
 			$this->getPageIdFromRecordIdentifiedInEditUrlArgument(),
 			$this->getPageIdFromContentObject(),
 			parent::getCurrentPageId(),
 		);
+		$this->currentPageId = 0; // parent::getCurrentPageId() set possible wrong value
```

Comment:
This line should be at line 59 instead so that right before the loop, we reset the internal variable. The reason for the problem if you have is that the loop never runs because the first check is immediately false (empty($this->currentPageId)). Rather than reset the value in this getter/resolver, which is bad, it should be reset right before determining the *new* active page UID.